Update AWS Batch min CPUs warning (#514)
seqeralabs/docs: platform_versioned_docs/version-22.4/compute-envs/aws-batch.mdx

- You can set **Min CPUs** to be greater than `0`, in which case some EC2 instances will remain active. An advantage of this is that pipeline executions will initialize faster.

  :::caution

>   Setting Min CPUs to a value greater than 0 will keep the required compute instances active, even when not running your pipeline executions. Note that this will result in additional AWS charges.

Reviewer: ```suggestion
  Setting Min CPUs to a value greater than 0 will keep the required compute instances active, even when your pipelines are not running. This will result in additional AWS charges.
```

Reviewer: Suggestion.
